it/stubs/APIGatewayAuthServiceStub.scala

```diff
@@ -5,15 +5,12 @@ import com.github.tomakehurst.wiremock.client.WireMock._
 object APIGatewayAuthServiceStub {
 
   def oauthRedirectSuccess(service: String): Unit = {
```

Comment:
I guess you can now remove the `service` parameter from this function as its no longer needed.